heartbeat/heartbeat.reference.yml

```diff
@@ -1349,6 +1349,9 @@ output.elasticsearch:
 # Name under which the template is stored in Elasticsearch
 #setup.template.json.name: ""
 
+# Set this option if the JSON template is a data stream.
+#setup.template.json.data_stream: false
```

Comment:
This setting needs to be added to the reference docs, too: https://github.com/elastic/beats/blob/master/libbeat/docs/template-config.asciidoc